### DISCUSSION TITLE
Error after --continue

### DISCUSSION THREAD

**Discussion by Nwilliams96** [ablab/spades#1184] [Q&A]
I am fairly certain my run has finished correctly, but I wanted it ensure that this is not an issue:

=== Error correction and assembling warnings:
 * 0:00:00.000     1M / 10M   WARN   StageManager             (stage.cpp                 : 158)   No saved checkpoint
======= Warnings saved to /scratch/u12034652_aad/clean.fastq.to.assemble/spades.assembly.output/402622_S27_output/warnings.log

**Comment by asl** [comment]
It's ok
